platform/openide.awt/src/org/openide/awt/ContextAction.java

```diff
@@ -284,28 +286,22 @@ public boolean equals(Object obj) {
         if (obj == this) {
             return true;
         }
-        
+
         if (obj instanceof ContextAction) {
             ContextAction c = (ContextAction)obj;
-            
+
             return type.equals(c.type) &&
                 selectMode.equals(c.selectMode) &&
                 performer.equals(c.performer) &&
                 Objects.equals(enableMonitor, c.enableMonitor);
         }
         return false;
     }
-    
-    private static final Reference<Object> NONE = new WeakReference<>(null);
```

Comment:
If NONE had been used consistently instead of null, then it would have been a useful way to avoid a lot of null checks. (Since you need a null check for the inner reference always but could avoid the outer one.) But it seems it was not really used this way.